src/scss/base.scss

```diff
@@ -59,21 +57,18 @@
 //    Styling for bare HTML elements (like H1, A, etc.). These come with
 //    default styling from the browser so we can redefine them here.
 @import "elements/asterisk";
-@import "elements/root";
 @import "elements/a";
 @import "elements/p";
 @import "elements/h1-6";
 @import "elements/body";
-@import "elements/img";
 @import "elements/ol-ul";
 
 
 // =====================================================================
 // 5. Objects
 //    Class-based selectors which define undecorated design patterns,
 //    for example media object known from OOCSS.
-@import "object/grid";
-@import "object/width-limiter";
+
```

Comment:
I think some extra space snuck in here :smile:

Reply:
fixed!

Reply:
got it. going to thumbs up stuff I fix